core/utils_test.py

```diff
@@ -66,7 +66,7 @@ def test_yaml_dict_conversion(self) -> None:
         test_dicts = [{}, {'a': 'b'}, {'a': 2}, {'a': ['b', 2, {'c': 3.5}]}]
```

Comment:
```suggestion
        test_dicts:List[Dict[str, Any]] = [
          {}, {'a': 'b'}, {'a': 2}, {'a': ['b', 2, {'c': 3.5}]}]
```
try this, mypy is unable to infer the type here so you will have to explicitly specify it